server.go

```diff
@@ -1278,6 +1304,9 @@ func (s *server) Stop() error {
 		s.invoices.Stop()
 		s.fundingMgr.Stop()
 		s.chanSubSwapper.Stop()
+		if s.towerClient != nil {
+			s.towerClient.Stop()
+		}
```

Comment:
Any reason for not doing this after disconnecting all peers? Not that big of a deal, but if we don't have any towers available to process a backup and shut down, then we can stall here a bit while the peer connections are still active when they don't need to be.

Reply:
good point, fixed